gu-provider/src/sync_exec.rs

```diff
@@ -110,7 +110,7 @@ error_chain!(
 
     errors {
         MailboxError(e : MailboxError){}
-        ExecutionError(exec: String, args: Vec<String>) {
+        ExecutionError(exec: String, args: Vec<String>, output: process::Output) {
             display("failed to execute command: {}, {:?}", exec, args)
```

Comment:
since you've added the output, use it when displaying this Error 
```suggestion
            display("failed to execute command: {}, {:?}, {:?}", exec, args, output)
```